webrtc-stats.html

```diff
@@ -1970,12 +1970,12 @@
               </dt>
               <dd>
                 <p>
-                  It is the total time each audio sample or video frame takes from the time it is
-                  received to the time it is rendered. The delay is measured from the time the
-                  first packet belonging to an audio/video frame enters the jitter buffer to the
-                  time the complete frame is sent for rendering after decoding. The average jitter
-                  buffer delay can be calculated by dividing the <code>jitterBufferDelay</code>
-                  with the <code>framesDecoded</code> (for video) or
+                  It is the total time in seconds each audio sample or video frame takes from the
```

Comment:
While we're here, can we say "It is the sum of the time, in seconds, that each audio sample or video frame spends from the time it is received to the time it is rendered, across all frames/samples."?
The fact that it's a sum isn't immediately obvious from the text.

Reply:
Done.